<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>generator</class>
 <widget class="QWidget" name="generator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>INS-GPS Generator</string>
  </property>
  <widget class="QLineEdit" name="sensors_total_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="data_folder_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_scenario_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Edit Scenario</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="generation_progress_bar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="scenario_open_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Scenario file</string>
   </property>
  </widget>
  <widget class="QLabel" name="sensors_total_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sensors total:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scenario_path_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="data_folder_select_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Results folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generate_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>140</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GENERATE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="status_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>330</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
